demo/src/features/mesh/shadow_map_resource.rs

```diff
@@ -370,30 +307,44 @@ fn calculate_shadow_map_views(
                 cube_map_view_directions.1,
             );
 
-            let near = 0.25;
+            let near = 0.1;
```

Comment:
You mentioned previous some visual artifacts, I've noticed that changing the near field can have a strong affect on precision in the depth buffer (even when we use the reverse z trick). It's *possible* if you still notice them, that this may be related. (I wouldn't expect it to be a problem though.)